jetty-core/jetty-server/src/main/config/etc/jetty-dos.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="insertHandler">
+    <Arg>
+      <New id="DosHandler" class="org.eclipse.jetty.server.handler.DoSHandler">
+        <Arg name="getId">
+          <Get>
+            <Name><Property name="jetty.dos.id.type" default="ID_FROM_REMOTE_ADDRESS"/></Name>
+            <Class><Property name="jetty.dos.id.class" default="org.eclipse.jetty.server.handler.DoSHandler"/></Class>
+          </Get>
+        </Arg>
+        <Arg name="rateFactory">
+          <New>
+            <Class><Property name="jetty.dos.rateControlFactory" default="org.eclipse.jetty.server.handler.DoSHandler$ExponentialMovingAverageRateControlFactory"/></Class>
+            <Arg name="samplePeriodMs" type="long"><Property name="jetty.dos.samplePeriodMs" default="-1"/></Arg>
+            <Arg name="alpha" type="double"><Property name="jetty.dos.alpha" default="-1.0"/></Arg>
+            <Arg name="maxRequestsPerSecond" type="int"><Property name="jetty.dos.maxRequestsPerSecond" default="100"/></Arg>
+          </New>
+        </Arg>
+        <Arg name="rejectHandler">
+          <New>
+            <Class><Property name="jetty.dos.rejectHandler" default="org.eclipse.jetty.server.handler.DoSHandler$DelayedRejectHandler"/></Class>
+            <Arg name="delayMs" type="long"><Property name="jetty.dos.delayMs" default="1000"/></Arg>
+            <Arg name="maxDelayQueue" type="int"><Property name="jetty.dos.maxDelayQueue" default="1000"/></Arg>
+            <Arg name="reject">
+              <New class="org.eclipse.jetty.server.handler.DoSHandler$StatusRejectHandler">
+                <Arg name="status"><Property name="jetty.dos.rejectStatus" default="429"/></Arg>
```

Comment:
I'd rename these properties as `jetty.dos.rejectHandler.delayed.delayMs` etc. so they are scoped to the RejectHandler.

Reply:
OK for delayMs, but the reject status will be common to all/most reject handlers